emf/common/xslt_engine/saxonpy_api.py

```diff
@@ -30,7 +69,6 @@ def xslt30_convert(source_file, stylesheet_file, output_file=None):
 
 def validate_xml(input_xml, schema_xml):
 
-    logger.info(f"Validating XML against XSD")
     if str == type(input_xml):
         document = etree.parse(input_xml)
```

Comment:
Please add here the same logic as with XSLT, eg it can be path or actual xml dock

Reply:
lxml etree.parse has this functionality already, it can take both path and file without issue